libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -927,6 +927,23 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info[] = {
     // @Description: Vertical position on screen
     // @Range: 0 21
     AP_SUBGROUPINFO(vtx_power, "VTX_PWR", 55, AP_OSD_Screen, AP_OSD_Setting),
+
+
+    // @Param: VTX_TEMP_EN
+    // @DisplayName: VTX_TEMP_EN
+    // @Description: Displays TEMP Power
+    // @Values: 0:Disabled,1:Enabled
+
+    // @Param: VTX_TEMP_X
+    // @DisplayName: VTX_TEMP_X
+    // @Description: Horizontal position on screen
+    // @Range: 0 59
+
+    // @Param: VTX_TEMP_Y
+    // @DisplayName: VTX_TEMP_Y
+    // @Description: Vertical position on screen
+    // @Range: 0 21
+    AP_SUBGROUPINFO(vtx_temp, "VTX_TEMP", 56, AP_OSD_Screen, AP_OSD_Setting),
```

Comment:
You can't just change the index of an existing parameter. This needs to go in an empty slot and there are only 64 per group so you need to pick a different group

Reply:
I went for index 63 which should be last in group.